src/main/java/com/plotsquared/holoplots/HoloPlotsPlugin.java

```diff
@@ -1,35 +1,85 @@
 package com.plotsquared.holoplots;
 
+import com.plotsquared.bukkit.util.BukkitUtil;
 import com.plotsquared.core.PlotAPI;
+import com.plotsquared.core.configuration.caption.Caption;
+import com.plotsquared.core.configuration.caption.LocaleHolder;
+import com.plotsquared.core.configuration.caption.TranslatableCaption;
+import com.plotsquared.core.plot.Plot;
+import com.plotsquared.core.util.PlayerManager;
+import com.plotsquared.core.util.task.TaskManager;
 import com.plotsquared.holoplots.config.Configuration;
+import com.plotsquared.holoplots.listener.ChunkListener;
+import com.plotsquared.holoplots.listener.PlotSquaredListener;
+import com.plotsquared.holoplots.provider.IHologramProvider;
+import com.plotsquared.holoplots.provider.impl.DecentHologramsProvider;
+import com.plotsquared.holoplots.provider.impl.HolographicDisplaysProvider;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.minimessage.Template;
 import org.bstats.bukkit.Metrics;
+import org.bstats.charts.SimplePie;
 import org.bukkit.Bukkit;
 import org.bukkit.Material;
 import org.bukkit.inventory.ItemStack;
 import org.bukkit.inventory.meta.SkullMeta;
 import org.bukkit.plugin.java.JavaPlugin;
 
 import java.io.File;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
 import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Function;
 
 public class HoloPlotsPlugin extends JavaPlugin {
 
+    public static final List<Caption> HOLOGRAM_CAPTIONS = List.of(
+            TranslatableCaption.of("signs.owner_sign_line_1"),
+            TranslatableCaption.of("signs.owner_sign_line_2"),
+            TranslatableCaption.of("signs.owner_sign_line_3"),
+            TranslatableCaption.of("signs.owner_sign_line_4")
+    );
     private static final int BSTATS_ID = 6402;
-    public static HoloPlotsPlugin THIS;
-    public static IHoloUtil HOLO = null;
+
+    private static final Map<String, Function<HoloPlotsPlugin, IHologramProvider>> SUPPORTED_PROVIDERS = Map.of(
+            "HolographicDisplays", HolographicDisplaysProvider::new,
+            "DecentHolograms", DecentHologramsProvider::new
```

Comment:
These Strings are currently duplicated, Can we move them to their corresponding classes as constants?